policy/policy.sample.json

```diff
@@ -0,0 +1,71 @@
+{
+    "SecurityPolicies": [
+        {
+            "Name": "policy1",
+            "PolicyID": "3710940D-FBC0-4A44-A95D-E549E628B4DF", # unique identifier across tenants, auto generated for POST.
+            "AppliedTo": [ # can attach multiple tenants to which the policy can be applied to.
+                {
+                    "Tenant":"tenant1", #optional if TenantID below is provided 
+                    "TenantID": "3A6E467F-370B-43EA-BCEC-C28912EDD752", # optional but good to have to relate things as names are not unique
+                    "Segment": "Segment1", # optional
+                    "SegmentID": "01CA06A0-429B-4EE1-9185-F17D5CE4D0F2",
+                },
+                {
+                    "Tenant":"tenant1",
+                    "Segment": "Segment2",
+                },
+            ],
+            "Tags": [
+                {
+                    "Key": "Platform",
+                    "Value": "Openstack",
+                },
+                {
+                    "Key": "TenantID",
+                    "Value": "D46FD18B-A381-4297-A6D4-9DA594EE71C8",
+                },
+            ], # meta data attached to policies for various external enviorments like openstack/kybernetes
+            "Direction" : "Egress", # can be Egress or Ingress.
+            "Rules": [
+                {
+                    "CidrBlock": "0.0.0.0/0", # IP from L3 header
```

Comment:
That's not what I had in mind. Instead:


    {
        "AppliedTo" : {
            "Tenant" : ...
            "Segment" : ...
        },
        "Peer" : {
            "CidrBlock" : "0.0.0.0/0"
        },
        "Direction" : "ingress",
        "Rules" : [
            {
                "Protocol" : "TCP",
                "DstPortRange" : ...
            },
            {
                "Protocol" : "UDP",
                "SrcPortRange" : ...
                "DstPortRange" : ...
            },
            {
                "Protocol" : "ICMP",
                "TYpeCode" : [ 8, 1 ]
            }
        ]
    }

As an aside: Note the UDP example where separate source and destination ports were applied.